tests/test_r2s.py

```diff
@@ -546,28 +547,45 @@ def test_r2s_script_step_by_step():
         thisdir, "files_test_r2s", "r2s_examples", "unstructured_r2s_run")
     _r2s_test_step1(r2s_run_dir, remove_step1_out)
     _r2s_test_step2(r2s_run_dir, remove_step1_out)
-
+    # test openmc r2s
+    try:
+        import openmc
+    except:
+        raise SkipTest
+    r2s_run_dir = os.path.join(
+            thisdir, "files_test_r2s", "r2s_examples", "openmc_r2s")
+    _r2s_test_step1(r2s_run_dir, remove_step1_out)
+    _r2s_test_step2(r2s_run_dir, remove_step1_out)
 
 def test_r2s_script():
+    # skip test without dagmc
+    try:
+        from pyne import dagmc
+    except ImportError:
+        raise SkipTest
 
-   # skip test without dagmc
-   try:
-       from pyne import dagmc
-   except ImportError:
-       raise SkipTest
-
-   remove_step1_out = False
-   r2s_run_dir = os.path.join(
-       thisdir, "files_test_r2s", "r2s_examples", "r2s_run")
-   _r2s_test_step1(r2s_run_dir, remove_step1_out)
-   _r2s_test_step2(r2s_run_dir, remove_step1_out)
-   # test sub-voxel r2s
-   r2s_run_dir = os.path.join(
-       thisdir, "files_test_r2s", "r2s_examples", "subvoxel_r2s_run")
-   _r2s_test_step1(r2s_run_dir, remove_step1_out)
-   _r2s_test_step2(r2s_run_dir, remove_step1_out)
-   # test unstructured r2s
-   r2s_run_dir = os.path.join(
-       thisdir, "files_test_r2s", "r2s_examples", "unstructured_r2s_run")
-   _r2s_test_step1(r2s_run_dir, remove_step1_out)
-   _r2s_test_step2(r2s_run_dir, remove_step1_out)
+    remove_step1_out = False
+    # test voxel r2s
+    r2s_run_dir = os.path.join(
+        thisdir, "files_test_r2s", "r2s_examples", "r2s_run")
+    _r2s_test_step1(r2s_run_dir, remove_step1_out)
+    _r2s_test_step2(r2s_run_dir, remove_step1_out)
+    # test sub-voxel r2s
+    r2s_run_dir = os.path.join(
+        thisdir, "files_test_r2s", "r2s_examples", "subvoxel_r2s_run")
+    _r2s_test_step1(r2s_run_dir, remove_step1_out)
+    _r2s_test_step2(r2s_run_dir, remove_step1_out)
+    # test unstructured r2s
+    r2s_run_dir = os.path.join(
+        thisdir, "files_test_r2s", "r2s_examples", "unstructured_r2s_run")
+    _r2s_test_step1(r2s_run_dir, remove_step1_out)
+    _r2s_test_step2(r2s_run_dir, remove_step1_out)
+    # test openmc r2s
+    try:
```

Comment:
Will raising `SkipTest` here indicate that we've skipped the r2s script test completely even though we've tested the other parts above?

Reply:
@pshriwise Yes. Tests above this "ShipTest" do not require openmc, they should be tested.

Reply:
Does our test output indicate this though? I'm just thinking that this could be confusing. The tests may still appear to be skipped entirely.

If someone new didn't have PyNE installed with `dagmc` support at first, and then installed `dagmc` but doesn't have `openmc` installed, then the test would still appear to be skipped entirely right?

Reply:
@pshriwise In the case you described, yes, the entire test will be skipped. The `HAVE_PYMOAB` is used to determine whether to skip these tests.

https://github.com/pyne/pyne/blob/9d18f546994d176b5a7d4f5ab3451656483e5b94/tests/test_r2s.py#L19

If the user didn't have PyNE installed with `dagmc`, then `moab` and `pymoab` is absent too. Therefore, all the tests in `test_r2s.py` will be skipped.